knowledge-base/kb-filedialogsdemos-notavailable.md

```diff
@@ -0,0 +1,37 @@
+---
+title: FileDialogs Demos Not Available in WPF Control Examples App
+description: "How to run FileDialogs WPF Demos application." 
+type: how-to
+page_title: How to run FileDialogs WPF Demos.
+slug: kb-filedialogsdemos-notavailable
+position: 0
+tags: filedialogs, demos
+ticketid: 1615001
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadFileDialogs for WPF</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to run `FileDialogs Demos` in `WPF Control Examples` application. 
```

Comment:
This should be describing the error. Something like "The control examples are not loading in the WPF Controls Examples application".